backend/package.json

```diff
@@ -24,11 +24,13 @@
     "nodemon": "2.0.19",
     "pino-pretty": "8.1.0",
     "ts-node": "10.8.2",
-    "tsc-alias": "1.6.11"
+    "tsc-alias": "1.6.11",
+    "@types/bcrypt": "5.0.0"
   },
   "dependencies": {
     "@fastify/static": "6.5.0",
     "dotenv": "16.0.1",
+    "bcrypt": "5.0.0",
```

Comment:
Looks like the same issue as described here https://github.com/BinaryStudioAcademy/bsa-2022-guruhub/pull/15#discussion_r939562920